projects/forge-angular/src/lib/proxy-components.module.ts

```diff
@@ -0,0 +1,157 @@
+import { ForgeSkeletonModule } from './skeleton';
+import { ForgeSliderProxyModule } from './slider';
+import { ForgeStepModule } from './step';
+import { ForgeStepperModule } from './stepper';
+import { ForgeSwitchProxyModule } from './switch';
+import { ForgeTabModule } from './tab';
+import { ForgeTabBarModule } from './tab-bar';
+import { ForgeTableModule } from './table';
+import { ForgeTextFieldModule } from './text-field';
+import { ForgeTimePickerProxyModule } from './time-picker';
+import { ForgeToastModule } from './toast';
+import { ForgeToolbarModule } from './toolbar';
+import { ForgeTooltipModule } from './tooltip';
+import { ForgeViewModule } from './view';
+import { ForgeViewSwitcherModule } from './view-switcher';
+
+export const proxyModules = [
+    ForgeAccordionModule,
+    ForgeAppBarModule,
+    ForgeAppBarHelpButtonModule,
+    ForgeAppBarMenuButtonModule,
+    ForgeAppBarNotificationButtonModule,
+    ForgeAppBarProfileButtonModule,
+    ForgeAppBarSearchModule,
+    ForgeAutocompleteProxyModule,
+    ForgeAvatarModule,
+    ForgeBackdropModule,
+    ForgeBadgeModule,
+    ForgeBannerModule,
+    ForgeBottomSheetModule,
+    ForgeBusyIndicatorModule,
+    ForgeButtonModule,
+    ForgeButtonToggleProxyModule,
+    ForgeButtonToggleGroupModule,
+    ForgeCalendarModule,
+    ForgeCardModule,
+    ForgeCheckboxModule,
+    ForgeChipProxyModule,
+    ForgeChipFieldModule,
+    ForgeChipSetModule,
+    ForgeCircularProgressModule,
+    ForgeColorPickerModule,
+    ForgeDatePickerProxyModule,
+    ForgeDateRangePickerProxyModule,
+    ForgeDialogModule,
+    ForgeDividerModule,
+    ForgeDrawerModule,
+    ForgeExpansionPanelModule,
+    ForgeFloatingActionButtonModule,
+    ForgeFilePickerModule,
+    ForgeIconModule,
+    ForgeIconButtonModule,
+    ForgeInlineMessageModule,
+    ForgeKeyboardShortcutModule,
+    ForgeLabelValueModule,
+    ForgeLinearProgressModule,
+    ForgeListModule,
+    ForgeListItemModule,
+    ForgeMenuModule,
+    ForgeMiniDrawerModule,
+    ForgeModalDrawerModule,
+    ForgeOpenIconModule,
+    ForgeOptionModule,
+    ForgeOptionGroupModule,
+    ForgePageStateModule,
+    ForgePaginatorModule,
+    ForgePopupProxyModule,
+    ForgeProductIconModule,
+    ForgeProfileCardModule,
+    ForgeQuantityFieldModule,
+    ForgeRadioModule,
+    ForgeRippleModule,
+    ForgeScaffoldModule,
+    ForgeSelectProxyModule,
+    ForgeSelectDropdownModule,
+    ForgeSkeletonModule,
+    ForgeSliderProxyModule,
+    ForgeStepModule,
+    ForgeStepperModule,
+    ForgeSwitchProxyModule,
+    ForgeTabModule,
+    ForgeTabBarModule,
+    ForgeTableModule,
+    ForgeTextFieldModule,
+    ForgeTimePickerProxyModule,
+    ForgeToastModule,
+    ForgeToolbarModule,
+    ForgeTooltipModule,
+    ForgeViewModule,
+    ForgeViewSwitcherModule
+];
+
+@NgModule({
+  imports: [ ...proxyModules ],
+  exports: [ ...proxyModules ]
+})
+export class ProxyComponentsModule {}
```

Comment:
Created this module as a parallel to the ForgeModule that includes all the proxy component modules.  They should potentially be combined in v3.